.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+<!-- Thank you for your contribution! The following items must be addressed before the code can be merged. Please don't hesitate to ask for help if you're unsure of how to accomplish any of the items. Feel free to remove check-list items that aren't relevant to your change -->
+
+  - [ ] Closes #xxxx .
+  - [ ] I am familiar with the [contributing guidelines](https://solarforecastarbiter-core.readthedocs.io/en/latest/contributing.html).
+  - [ ] Tests added.
+  - [ ] Updates entries to [`docs/source/api.rst`](https://github.com/SolarArbiter/solarforecastarbiter-core/blob/master/docs/source/api.rst) for API changes.
+  - [ ] Adds change descriptions to appropriate "whats new" in [`docs/source/whatsnew`](https://github.com/SolarArbiter/solarforecastarbiter-core/tree/master/docs/source/whatsnew) for all changes.
+  - [ ] New code is fully documented. Includes [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) compliant docstrings, examples, and comments where necessary.
+  - [ ] Issue and pull request are tagged with relevant labels and milestone.
```

Comment:
Not every PR has an issue. It would help to have an example here to show text and formatting.  I think assigning milestones is up to maintainers rather than contributors.

Reply:
True, this is a maintainer activity. I'd like to have some kind of reminder for maintainers before merging, though. Maybe we could have a `## Contributor` section and a `## Maintainer` section. Of course, the maintainer should double check everything in the contributor section.

What text and formatting are you referring to?

Reply:
The actual formatting for adding issue and pull request tags as an example. Something like `(:pull:`14`, :issue:`41`)`

Reply:
Oh, yes, I think that would be good to add to the whats new line. This checkbox is supposed to remind maintainers to use GitHub's assign label/milestone features. This would have really helped me when writing the last quarterly report. I'm less concerned about that for pvlib, but doesn't hurt.

To be clear, I'm referring to these items for this PR...

<img width="666" alt="Screen Shot 2019-08-07 at 9 42 56 AM" src="https://user-images.githubusercontent.com/4383303/62641173-e599b700-b8f7-11e9-8126-44184ee2f497.png">

Reply:
Maybe there is a nice Github app that can make it so labels/milestones are added before a merge

Reply:
Oh, I read "tagged" and assumed it meant the embedded text link.  Maybe "Appropriate GitHub Labels and Milestone are assigned to the Pull Request"    I'd add the reminder to assign Labels to an Issue to the issue template.

Reply:
We might be able to extend https://github.com/stilliard/github-task-list-completed to do this if something doesn't already exist

Reply:
I might keep "Issue and pull request are tagged..." here and also add the issue reminder to the issue template. This will probably help use catch missing labels since the issue will likely be automatically closed once the PR is merged.